src/app/configs/data/shortcuts.xml

```diff
@@ -548,11 +548,11 @@
     </SC>
   <SC>
     <key>add-hairpin</key>
-    <seq>Shift+,</seq>
+    <seq>&gt;</seq><!-- but see also 'next-beat-TEXT' -->
     </SC>
   <SC>
     <key>add-hairpin-reverse</key>
-    <seq>Shift+.</seq>
+    <seq>&lt;</seq><!-- but see also 'prev-beat-TEXT' -->
```

Comment:
It was this in Mu3, I see no reason whatsoever not to change it back, this is the most natural way to do add a diminuendo